hook/vaas/hook.go

```diff
@@ -125,17 +127,38 @@ func (sh *Hook) RegisterBackend(taskInfo mesosutils.TaskInfo) error {
 	return nil
 }
 
+func (sh *Hook) getFrontendPort(ports []mesos.Port, director string) (*mesos.Port, error) {
```

Comment:
I would rather see this part inside mesosutils.TaskInfo.

Finding labeled port, would be a good idea for other hooks too.